### PR TITLE
Differentiate Service Account, Service Registries and Kafka Instance names by including cloud provider in name.

### PR DESCRIPTION
**Pull request by tonyxrmdavidson**
Update to rename Kafka Instances, Service Accounts and Service Registry using the CLOUD_PROVIDER environment variable. This is needed to distinguish between aws staging names and gcp staging names and will avoid errors when running concurrent E2E test runs.

### CODE REVIEW

File: src/test/java/io/managed/services/test/registry/RegistryKafkaIntegrationTest.java [hunk 1]
```diff
@@ -60,9 +60,9 @@
 public class RegistryKafkaIntegrationTest extends TestBase {
     private static final Logger LOGGER = LogManager.getLogger(RegistryKafkaIntegrationTest.class);
```
Comment (henryZrncik) [inline src/test/java/io/managed/services/test/registry/RegistryKafkaIntegrationTest.java:62]: currently only place for 1 service regisstry instance, one test will fail at the moment.

**Review by henryZrncik — CHANGES_REQUESTED** [review]
just few changes: 
firstly forgotten class (e.g. new class QuotaTest), althoguh this class will actually need more diligent treatment. 

secondly: service registries: currently 1 instnace so running this will be blocked untiil resolved. 

thirdly: there are also other resources beside those which had launch key, mainly resources on user OSD, so there are lot of resources in quickstarts which will clash unless you add it there.

**Comment by agullon** [comment]
Hi @tonyxrmdavidson , I think https://github.com/bf2fc6cc711aee1a0c2a/e2e-test-suite/blob/concurrentGcpAwsStage/src/test/java/io/managed/services/test/kafka/KafkaOverseeingTest.java is missing, can you take a look at it? thanks

**Comment by tonyxrmdavidson** [comment]
Do Not Merge WIP

**Review by henryZrncik — CHANGES_REQUESTED** [review]
suffixes works for me, 

what is worth considering now is the fact that in two commits you are keeping log information for stage dev users, in a file that is not even in the final version. So I would suggest squash of these commits into the 1. 

also considering the name, of these PR, it suggests that AWS, GCP can run next to each other, for that you need to do more changes in code base (mainly modify quota Class), and also parametrize endpoints in billing metrics class (currently we are targeting data plane for aws). 

So we can either push this but renaming it to something more suitable, or address also points mentioned above.

**Comment by henryZrncik** [comment]
Anyway, if you want to proceed with running it next to each other, least painful solution right now would be to create suite which would omit  quota tests, billing metric tests, for gcp. (and resolve this on their own, but these are not so drastic changes anyway). 
if so, aws suites continues to test all classes as it is doing at this moment.

**Review by henryZrncik — APPROVED** [review]
with these changes works for me.

**Review by kornys — APPROVED** [review]

**Pull request merged**